core/app/helpers/spree/base_helper.rb

```diff
@@ -1,7 +1,7 @@
 module Spree
   module BaseHelper
     def available_countries
-      checkout_zone = Spree::Zone.find_by(name: Spree::Config[:checkout_zone])
+      checkout_zone = current_store.zone || Spree::Zone.find_by(name: Spree::Config[:checkout_zone])
```

Comment:
```suggestion
      checkout_zone = current_store.zone || Spree::Zone.default_checkout_zone
```

let's add `default_checkout_zone` method to `Zone` model please

Reply:
@damianlegawiec should I treat this `No Limits` zone like a default if `Spree::Config[:checkout_zone]` not set ?

Reply:
@damianlegawiec fixed, I add `No Limits` zone like a default, please let me know if I should change